types/react-sortable-tree/react-sortable-tree-tests.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+import SortableTree,
+    {
+        SortableTreeWithoutDndContext,
+        defaultGetNodeKey,
+        NodeRenderer,
+        TreeItem,
+        defaultSearchMethod,
+        SearchData,
+        NodeData,
+        ExtendedNodeData,
+        FullTree,
+        OnVisibilityToggleData,
+        PreviousAnNextLocation
+    } from "react-sortable-tree";
+import { ListProps, ListRowRenderer } from "react-virtualized";
+
+class Test extends React.Component<void, void> {
```

Comment:
`React.Component<void, void>` => can be simplified to `React.Component` thx to #17288